index.html

```diff
@@ -27,15 +27,18 @@
 			<div id="rectangle"></div>
 			<p>The visualization of the database of the Syrian Heritage Archive aims to ensure that Syrian cultural heritage is not forgotten. Thanks to digitisation of databases, it may act as a basis for any prospective reconstruction.</p>
 		</section>
-		<section id="kalli"><img src="http://placehold.it/150x50/1E1938/fff" alt="Foo-Bar-Institut"></section>
+		<section id="kalli"><img src="assets/anfang_.png" alt="begin" height="59px" width="62px"></section>
 	</header>
 	<nav id="filters"><ul></ul></nav>
 	<main id="gallery"></main>
 	<footer id="footer">
-		<img src="http://placehold.it/150x50/1E1938/fff" alt="Foo-Bar-Institut">
-		<img src="http://placehold.it/50x50/1E1938/fff" alt="Foo-Bar-Institut">
-		<img src="http://placehold.it/100x70/1E1938/fff" alt="Foo-Bar-Institut">
-		<img src="http://placehold.it/300x50/1E1938/fff" alt="Foo-Bar-Institut">
+		<section id="kalli"><img src="assets/ende_.png" alt="begin" height="59px" width="62px"></section>
```

Comment:
I guess `alt="begin` should be `alt="end"`.

Also, you told me this is an arabic letter/symbol. Why not use this as an `alt` text?